chapter/8/big-data.md

```diff
@@ -1,11 +1,229 @@
 ---
 layout: page
 title:  "Large Scale Parallel Data Processing"
-by: "Joe Schmoe and Mary Jane"
+by: "Jingjing and Abhilash"
 ---
+## Introduction
+`JJ: Placeholder for introduction` The booming Internet has generated big data...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. {% cite Uniqueness --file big-data %}
 
-## References
+This chapter is organized in <label for="note1" class="margin-toggle sidenote-number"></label><input type="checkbox" id="note1" class="margin-toggle"/><span class="sidenote">JJ: need to fill in more stuff</span>
+
+- **Data paralleling**:
+  - MapReduce {% cite dean2008mapreduce  --file big-data %}
+  - FlumeJava {% cite chambers2010flumejava --file big-data %}
+  - ...
+- **Graph paralleling**:
+  - Pregel
+  - ...
+
+For each programming model, we will discuss the motivation, basic model, execution model, fault-tolerance and performance.
```

Comment:
Why would you introduce each programming model one by one? It's better to make categories, like "data parallelism" and then to put systems with data parallel programming models into that category.

Also, there are (1) programming models and (2) underlying models of execution. These are hugely important dimensions that need to be treated separately. So far you've lumped everything all together and you're just making a list devoting a little paragraph to each model and not really making this distinction at all. 

Instead you should probably have subsections devoted to programming models and execution models (e.g., spark executes in-memory while Hadoop/MR writes intermediate data to disk)